doc/api/crypto.md

```diff
@@ -1908,14 +1908,20 @@ const {
 ### `keyObject.asymmetricKeyDetails`
 <!-- YAML
 added: v15.7.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/39851
+    description: Expose `RSASSA-PSS-params` sequence parameters
+                 for RSA-PSS keys.
 -->
 
 * {Object}
-  * `modulusLength`: {number} Key size in bits (RSA, DSA).
-  * `publicExponent`: {bigint} Public exponent (RSA).
+  * `modulusLength`: {number} Key size in bits (RSA, RSA-PSS, DSA).
+  * `publicExponent`: {bigint} Public exponent (RSA, RSA-PSS).
```

Comment:
I would prefer to revert these two lines @panva because I believe they are not referring to the key type, which we usually write as `'rsa'`, `'rsa-pss'`, etc., but to the algorithm RSA. The parameters have different meaning for RSA and DSA, but the meaning is the same for RSA and RSA-PSS.

How it works in my head:

```
+-------------+
|             |
|  +-------+  |
|  |     <-------- modulusLength, publicExponent
|  |  RSA  |  |
|  +-------+  |
|             |
|           <------------- hashAlgorithm, mgf1HashAlgorithm, saltLength
|   RSA-PSS   |
|             |
+-------------+
```

Reply:
Fair enough.